src/component/Tooltip.tsx

```diff
@@ -115,7 +115,7 @@ function TooltipInternal<TValue extends ValueType, TName extends NameType>(props
   const viewBox = useViewBox();
   const accessibilityLayer = useAccessibilityLayer();
   const { active: activeFromContext, payload: payloadFromProps, coordinate, label } = useTooltipContext();
-  // TODO this will fail tests until Area, Bar, and Line all push their own payloads
+  // TODO this will fail tests until Tooltip props and tooltipAxis are in Redux
```

Comment:
What even is tooltipAxis? That seems like something we should rename? Is it an axis or is it tooltip information?

Reply:
One of the axes is selected based on layout and dice roll and that's treated as more important than others. I'm following what was already there.
Seems to me that most people only use one axis anyway so this doesn't cause too many problems? We can look into it... Later.

Reply:
Ah right. We can use axisIds to match axes the same as before though yeah?

Reply:
I am planning to use the exact same logic as was in generateCategoricalChart, only move it to Redux so that it no longer requires direct DOM access.